File: Cpp/Program/TokoElektronik.cpp
#include "Produk.cpp"

class TokoElektronik : public Produk {
private:
    int idToko;
    string namaToko;
    string pemilikToko;
    string alamatToko;
    string noIzinUsaha;
    int stok;

public:
    TokoElektronik() {
        idToko = 0;
        namaToko = "";
        pemilikToko = "";
        alamatToko = "";
        noIzinUsaha = "";
        stok = 0;
    }

    void setIdToko(int id) { idToko = id; }
    void setNamaToko(string n) { namaToko = n; }
    void setPemilikToko(string p) { pemilikToko = p; }
    void setAlamatToko(string a) { alamatToko = a; }
    void setNoIzinUsaha(string no) { noIzinUsaha = no; }
    void setStok(int s) { stok = s; }

    int getIdToko() { return idToko; }
    string getNamaToko() { return namaToko; }
    string getPemilikToko() { return pemilikToko; }
    string getAlamatToko() { return alamatToko; }
    string getNoIzinUsaha() { return noIzinUsaha; }
    int getStok() { return stok; }
};

File: Java/Program/TokoElektronik.java
public class TokoElektronik extends Produk {
    private int idToko, stok;
    private String namaToko, pemilikToko, alamatToko, noIzinUsaha;

    // Tambahkan konstruktor kosong
    public TokoElektronik() {}

    // Getter dan Setter
    public int getIdToko() { return idToko; }
    public void setIdToko(int v) { idToko = v; }

    public String getNamaToko() { return namaToko; }
    public void setNamaToko(String v) { namaToko = v; }

    public String getPemilikToko() { return pemilikToko; }
    public void setPemilikToko(String v) { pemilikToko = v; }

    public String getAlamatToko() { return alamatToko; }
    public void setAlamatToko(String v) { alamatToko = v; }

    public String getNoIzinUsaha() { return noIzinUsaha; }
    public void setNoIzinUsaha(String v) { noIzinUsaha = v; }

    public int getStok() { return stok; }
    public void setStok(int v) { stok = v; }
}